tests/test_api.py

```diff
@@ -0,0 +1,25 @@
+from tests import SiteTest, app
+
+class ApiEndpointsRootEndpoints(SiteTest):
```

Comment:
doesn't flake8 demand two newlines before this?

Reply:
skipping